code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -266,6 +266,15 @@
 	new /obj/item/ammo_storage/speedloader/a357(src)
 	return
 
+obj/item/weapon/storage/box/syndie_kit/cheaptide
+	name = "box (CT)"
+
+/obj/item/weapon/storage/box/syndie_kit/cheaptide/New()
+	..()
+	new /obj/item/weapon/implanter/traitor(src)
+	new /obj/item/clothing/glasses/sunglasses/sechud/syndishades(src)
+	return
```

Comment:
no trailing returns (yes I know you were just copying oldcode)

Reply:
Fixed, assuming that means ''delete that return''.